docker_image_cleaner/cleaner.py

```diff
@@ -192,13 +197,52 @@ def main():
                     time.sleep(max(delay_seconds, 30))
                     continue
 
-                toc = time.perf_counter()
                 deleted = pruned[key]
+                # pruned looks like:
+                # {
+                #     "ImagesDeleted": [
+                #         {"Deleted": "sha256:5611ea8655"},
+                #     ],
+                #     "SpaceReclaimed": 4563228463,
+                # }
```

Comment:
:heart: I appreciate this soooooo much, its otherwise one part of review / maintenance / debugging that can take a long time to figure out